libs/partners/anthropic/tests/integration_tests/test_chat_models.py

```diff
@@ -340,12 +389,15 @@ def test_tool_use() -> None:
 
     # Test streaming
     first = True
+    chunks = []  # type: ignore
```

Comment:
did you mean to add something else here operating on chunks? Could we just `assert not first` instead of chunk length otherwise or remove change?

Reply:
It's checking to make sure we get multiple message chunks via the assert below.

Reply:
It's checking to make sure we get multiple message chunks via the assert below